INSTALL.rst

```diff
@@ -1,6 +1,249 @@
+
+After your installation, you should be able to run the following code
+from a python console or jupyter notebook:
+
+.. code-block:: python
+
+  import skimage
+  print("scikit-image installed with version " + skimage.__version__)
+
+Should you have any questions getting started using ``scikit-image`` after
+reading this guide, please open an issue on
+`github <https://github.com/scikit-image/scikit-image/issues>`_
+to get additional help.
+
+For instructions on how to install scikit-image from source, please see the
+contribution guide.
+.. TODO: cross reference the contribution guide.
+
+1. Scientfic Python Distributions
+---------------------------------
+
+``scikit-image`` comes pre-installed with several Python distributions
+including:
+
+- `Anaconda <https://www.anaconda.com/distribution/>`_
+- `Python(x,y) <https://python-xy.github.io/>`_
+- `WinPython <https://winpython.github.io/>`_
+
+These have the advantage of comming pre-installed with many other useful
+packages for scientific computing and image analysis. Follow the instructions
+on the respective websites to get going.
+
+Note that the version of ``scikit-image`` installed with these distributions
+may different than the one you expect. This should not stop you from getting
+started with your project. We simply suggest you check the installed version of
+``scikit-image`` with the code snippet included at the top of this guide.
+
+
+2. ``pip`` based installation
+-----------------------------
+
+To install ``scikit-image`` with `pip <https://pip.pypa.io/en/stable/>`_, you
+may use the following commands
+
+.. code-block:: bash
+
+  # Update pip to a more recent version
+  python3 -m pip install -U pip
+  # Install scikit-image
+  python3 -m pip install -U scikit-image
+
+
+This is the easiest way of obtaining the latest released version of
+``scikit-image``.
+
+We also offer an easy way to install related packages. These are often included
+in scientific python distributions. To install many of them using ``pip`` use
+the command:
+
+.. code-block:: bash
+  python3 -m pip install scikit-image[optional]
+
+
+A note on virtual environments
+++++++++++++++++++++++++++++++
+
+Scientific Python Distributions listed in the first section take care to only
+install compatibile packages together. Often conflicts can arise between
+specific combinations of packages. To avoid breaking your python environments,
+we recommendg learning about virtual environments in python. You can learn more
+about them by reading Python's official documentation on `Virtual Environments
+and Packages <https://docs.python.org/3/tutorial/venv.html>`_.
+
+
+Known challenges
+++++++++++++++++
+
+1. Different python installations can cause confusion. To check which python
+   executable you are using, use the commands:
+
+
+.. code-block:: bash
+  # Linux and MAC
+  which python3
+  # Windows
+  where python3
+
+From your python console or jupyter notebook, you may use the python commands:
```

Comment:
Also, I feel a little pedantic for raising this :sweat_smile:, I think "Python" and other names like "Jupyter" should be capitalized.

Reply:
It's already done in a few places so maybe it's just due to being WIP.

Reply:
No thanks for clarifying that this is what you preferred